src/android/Print.java

```diff
@@ -259,6 +259,9 @@ private boolean createReceiptData(Order order) {
 			if (order.customer_address != null && !order.customer_address.isEmpty())
 				builder.append("Address: " + order.customer_address + "\n");
 
+			if (order.handoff_provider != null && !order.handoff_provider.isEmpty())
+				builder.append("Marketplace Provider: " + order.handoff_provider + "\n");
```

Comment:
Same question/comment here as the other PR. Are we sure this order will have a `handoff_provider` field and not a `third_party_marketplace` field, which is the version returned in our API?